build/configs/default_config.yaml

```diff
@@ -154,5 +154,8 @@ debug:
   # pprof for debugging
   pprof: false
 
+  # whether to mask errors returned in p2p responses for security
+  maskErrors: false
```

Comment:
That shouldn't be an option. Options that make a product less secure are bad options.